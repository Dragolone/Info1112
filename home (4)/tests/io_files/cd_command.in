pwd
cd ..
pwd
cd /nonexistent_dir
pwd
exit
