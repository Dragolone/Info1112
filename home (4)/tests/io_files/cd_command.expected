/home/tests
/home
cd: no such file or directory: /nonexistent_dir
/home
